sunbird-rc-esignet-integration-impl/src/main/java/io/mosip/esignet/sunbirdrc/integration/service/SunbirdRCAuthenticationService.java

```diff
@@ -44,22 +43,22 @@
 @Slf4j
 public class SunbirdRCAuthenticationService implements Authenticator {
 
-    private final String filterOperator="eq";
+    private final String Filter_Operator="eq";
```

Comment:
if this is constant use FILTER_EQUALS_OPERATOR

Reply:
changed